### PR TITLE
Added IPython

### PR DESCRIPTION
**Pull request by pelson**
I'm working on a few of the Jupyter project libraries - this being the first of them.

Ping @minrk as I've named you as a maintainer. Is that OK?

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks for doing this @pelson.

**Comment by pelson** [comment]
So the linter is good, and everything builds as expected, so I'll go ahead and merge. @minrk - if there are any review actions, please feel free to add them here or as issues in the feedstock.

**Comment by jakirkham** [comment]
Also, if there are other people that you, @minrk, that you think should be added, please let us know or feel free to add them in the feedstock.

File: recipes/ipython/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pickleshare
+    - simplegeneric >0.8
+# These are listed in the setup.py, but do not cause the build tests to fail.
+# In order to enable these dependencies, additional tests will need to be defined.
```
Comment (minrk) [inline recipes/ipython/meta.yaml]: These commented-out dependencies are recent additions, not yet in a release.

Comment (pelson) [inline recipes/ipython/meta.yaml]: Aha! Sorry, should have been paying attention to the release, not master.
At least they are nicely teed up for next release :wink:

**Pull request merged**